README.md

```diff
@@ -46,24 +46,34 @@ ros2 run performance_test perf_test --help
 ```
 # Plot results
 
+To plot the results, you will need to install the perfplot tool from the apex_performance_plotter python module.
+
+```bash
+pip3 install performance_test/helper_scripts/apex_performance_lotter
```

Comment:
There is a typo in the name of the script here should be `apex_performance_plotter` instead of `apex_performance_lotter`

Reply:
@pbaughman Can we also please mention from where should we run this command?

Reply:
@daggarwa I'm fixing this so it can be run from anywhere.

Reply:
@pbaughman Okay thanks. That sounds awesome!